TelstarLogistics/RouteplannerTests/DjikstraTests.cs

```diff
@@ -0,0 +1,57 @@
+
+
+using Humanizer;
+
+namespace RouteplannerTests
+{
+    [TestClass]
+    public class DjikstraTests
+    {
+        [TestMethod]
+        public void TestFindShortestPath()
+        {
+            //0 connect to 2
+            //1 connect to 2
+            //2 connect to 0 and 1
+            int[,] graph = new int[,]
+            {
+                { 0, 0, 2 }, //0 connections
+                { 0, 0, 2 }, //1 ----||----
+                { 2, 2, 0 }, //2 ----||----
+            };
+            int sourceNode = 0;
+            int destinationNode = 1;
+
+
+
+            int[] distances = TelstarLogistics.Services.RoutePlanning.Dijkstra.ComputePaths(graph, sourceNode, out var parents);
+
+            Assert.AreEqual(4,distances[destinationNode]);
+        }
+
+        [TestMethod]
+        public void TestRetracePath()
+        {
+            //0 connect to 2
+            //1 connect to 2
+            //2 connect to 0 and 1
+            int[,] graph = new int[,]
+            {
+                { 0, 0, 2 }, //0 connections
+                { 0, 0, 2 }, //1 ----||----
+                { 2, 2, 0 }, //2 ----||----
+            };
+            int sourceNode = 0;
+            int destinationNode = 1;
+
+
+
+            int[] distances = TelstarLogistics.Services.RoutePlanning.Dijkstra.ComputePaths(graph, sourceNode, out var parents);
+            var path = TelstarLogistics.Services.RoutePlanning.Dijkstra.GetPath(destinationNode, parents);
+            Assert.IsTrue(path[0] == 0 && path[1] == 2 && path[2] == 1);
+        }
+    }
+
```

Comment:
empty white spaces

Reply:
general note: there are multiple empty lines in your PR if you could run through all files